src/langcheck/metrics/__init__.py

```diff
@@ -13,6 +14,8 @@
                                               is_json_array, is_json_object,
                                               matches_regex, validation_fn)
 
+_model_manager = ModelManager()
```

Comment:
Instead of initializing this here, what do you think about creating a separate directory inside of `metrics`? E.g. something like
```
metrics/
    model_manager/
        __init__.py
        _model_loader.py
        _model_management.py
``` 
and in the model_manager's init file, we run
```
_model_manager = ModelManager()
```

Since not all metrics require running models (e.g. the `text_structure` metrics)